var/spack/repos/builtin/packages/nim/package.py

```diff
@@ -132,6 +130,22 @@ def append_rpath(path, libdirs):
         if spec.satisfies("+sqlite"):
             append_rpath("lib/wrappers/sqlite3.nim", spec["sqlite"].libs.directories)
 
+        # Musl defines SysThread as a struct *pthread_t rather than an unsigned long as glibc does.
+        if self.spec.satisfies("^[virtuals=libc] musl"):
+            if self.spec.satisfies("@devel,1.9.3:"):
+                pthreadModule = "lib/std/private/threadtypes.nim"
+            elif self.spec.satisfies("@=0.19.6"):
```

Comment:
This could be:
```suggestion
            elif self.spec.satisfies("@:0.19"):
```
ref: https://github.com/nim-lang/Nim/commit/228d53006a96c97d1f569e288cda7197a993868d

Reply:
It doesn't apply to the packaged in-dev version `@0.19.9`, but I can change it to `@:0.19.6`.
https://github.com/spack/spack/blob/cb9c9e6b8ce78c32609920c65072ffe344d92d63/var/spack/repos/builtin/packages/nim/package.py#L49-L53